test/functional/framework.go

```diff
@@ -23,6 +24,24 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+const (
+	applicationLog = "application"
+	auditLog       = "audit"
+	k8sAuditLog    = "k8s"
+)
+
+var fluentdLogPath = map[string]string{
+	applicationLog: "/var/log/containers",
+	auditLog:       "/var/log/audit",
+	k8sAuditLog:    "/var/log/kube-apiserver",
```

Comment:
I don't know it matters for this test but i think you are missing the openshift apiserver audit logs here